src/WP_CLI/CommandWithTranslation.php

```diff
@@ -91,6 +91,10 @@ public function update( $args, $assoc_args ) {
 			$num_to_update += count( $available_updates );
 
 			if ( ! Utils\get_flag_value( $assoc_args, 'dry-run' ) ) {
+				if ( ! is_array( $available_updates ) ) {
```

Comment:
This needs to be before the `$num_to_update` line to fix the "Warning: count(): Parameter must be an array or an object that implements Countable" warning too.

Reply:
You're right. Didn't see that!